NEXT-STEPS.md

```
@@ -176,35 +200,35 @@ Specs 048–051 are now implemented. The next step is to run a new reproduction
+    uv run python scripts/evaluate_selective_prediction.py --input <both_run.json> --mode few_shot --confidence hybrid_consistency
+    ```
+
+   Notes:
+   - `token_msp|token_pe|token_energy` variants only work when the backend returns token logprobs; if the run artifact doesn’t include `item_signals[*][\"token_*\"]`, those variants will fail fast (by design).
+   - If `token_*` keys are present, also evaluate:
+
+     ```bash
+     uv run python scripts/evaluate_selective_prediction.py --input <both_run.json> --mode few_shot --confidence token_msp
+     uv run python scripts/evaluate_selective_prediction.py --input <both_run.json> --mode few_shot --confidence token_pe
+     uv run python scripts/evaluate_selective_prediction.py --input <both_run.json> --mode few_shot --confidence token_energy
+     ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)